geth/geth.sh

```diff
@@ -2,77 +2,48 @@
 
 set -exu -o pipefail
 
+source /config/vars.env
+
 VERBOSITY=${GETH_VERBOSITY:-4}
 GETH_DATA_DIR=/db
-GETH_KEYSTORE_DIR="$GETH_DATA_DIR/keystore"
-GETH_CHAINDATA_DIR="$GETH_DATA_DIR/geth/chaindata"
-GENESIS_FILE_PATH="${GENESIS_FILE_PATH:-/genesis.json}"
-BLOCK_SIGNER_PRIVATE_KEY="45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8"
-#BLOCK_SIGNER_ADDRESS="0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b"
-RPC_PORT="${RPC_PORT:-8545}"
-WS_PORT="${WS_PORT:-8546}"
-BOOTNODE_KEY_HEX=${BOOTNODE_KEY_HEX:-65f77f40c167b52b5cc70fb33582aecbdcd81062dc1438df00a3099a07079204}
-NETWORKID=69
-ENABLE_MINING=true
+GETH_GENESIS=/shared-configs/genesis.json
```

Comment:
It could go either way. This works now because the `generated-genesis.json` is a copy of `/shared-configs/genesis.json`. Actually `/shared-configs/genesis.json` is the master file which is located on the lcli testnet directory.

One issue I'm noticing with docker is that bind mounts take a couple seconds to synchronize bind mounts. And `generated-genesis.json` is a bind-mount to the lcli testnet directory. So using the testnet directory here rather than `generated-genesis.json` prevents a race condition for the mount sync.